x/lockup/client/cli/tx.go

```diff
@@ -72,3 +73,20 @@ func NewSetRewardReceiverAddress() (*osmocli.TxCliDesc, *types.MsgSetRewardRecei
 		Long:  "sets reward receiver address for the designated lock id",
 	}, &types.MsgSetRewardReceiverAddress{}
 }
+
+// NewRebondTokensCmd allows a user to rebond their tokens if they have unlocked them.
+func NewRebondTokensCmd() (*osmocli.TxCliDesc, *types.MsgRebondTokens) {
+	return &osmocli.TxCliDesc{
+		Use:   "rebond-tokens",
```

Comment:
note: no hardcoded parameters since we changed it to be parsed dynamically from struct's fields

Reply:
if we add [lock-id] here, it will look like we need to provide 2 ids, something like: `rebond-tokens [lock-id] [id]`